python/paddle/jit/dy2static/call_transformer.py

```diff
@@ -17,6 +17,7 @@
 from paddle.utils import gast
 
 from .base_transformer import BaseTransformer
+from .utils import is_builtin
```

Comment:
可以改成这样，跳过import-but-unused的检查，参考：https://flake8.pycqa.org/en/2.6.0/warnings.html

```suggestion
from .utils import is_builtin   # noqa: F401
```